recipes/default.rb

```diff
@@ -0,0 +1,36 @@
+include_recipe 'apt'
+
+type = case node['platform_family']
+       when 'fedora', 'rhel'
+         'rpm'
+       else
+         'deb'
+       end
+
+packagecloud_repo 'StackStorm/stable' do
+  type type
+end
+
+package 'st2mistral'
+
+# ensure directories are managed
+[
+  node['openstack-mistral']['etc_dir']
+].each do |path|
+  directory "creating mistral directory #{path}" do
```

Comment:
> down stream cookbooks can overwrite that value

What is a use case for this?  The only time i have seen this is when you use `/opt` instead, but thats only when the packaging uses `/opt` too.